---
title: '&#39;Throw&#39; deyimi dışında işleneni atlayın olamaz bir &#39;Catch&#39; deyimi veya iç bir &#39;son&#39; deyimi'
ms.date: 07/20/2015
f1_keywords:
- vbc30666
- bc30666
helpviewer_keywords:
- BC30666
ms.assetid: a208a6ea-0e36-4bf1-8984-4de1a0e38a2a
ms.openlocfilehash: e263e98cce568deb5597c4b03c1b8b6534d6b0d3
ms.sourcegitcommit: 3d5d33f384eeba41b2dff79d096f47ccc8d8f03d
ms.translationtype: MT
ms.contentlocale: tr-TR
ms.lasthandoff: 05/04/2018
---
# <a name="39throw39-statement-cannot-omit-operand-outside-a-39catch39-statement-or-inside-a-39finally39-statement"></a>&#39;Throw&#39; deyimi dışında işleneni atlayın olamaz bir &#39;Catch&#39; deyimi veya iç bir &#39;son&#39; deyimi
`Throw` deyimleri dışında `Catch` deyimi bir özel durum nesnesi adı sağlamanız gerekir.  
  
 **Hata Kimliği:** BC30666  
  
## <a name="to-correct-this-error"></a>Bu hatayı düzeltmek için  
  
1.  Türetilen bir özel durum nesnesinin adını belirleyin `System.Exception`.  
  
2.  Kodunuzu yeniden yapılandırmayı böylece `Throw` açıklamadır içinde bir `Catch` bloğu.  
  
## <a name="see-also"></a>Ayrıca Bkz.  
 [Throw Deyimi](../../visual-basic/language-reference/statements/throw-statement.md)  
 [Try...Catch...Finally Deyimi](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)  
 [Visual Basic'te özel durum sınıfı](http://msdn.microsoft.com/library/9aac396f-34ca-4afb-8e6c-e523cb690ba9)  
 [Özel durum ve hata Visual Basic'te işleme](http://msdn.microsoft.com/library/3e351e73-cf23-40ab-8b60-05794160529e)
